### ISSUE TITLE
keepalive not working

### ISSUE THREAD

**Issue by andreas-globi** [chrome-php/headless-chromium-php#637]
I've read some other issues here but they're all referring to symphony, which I'm not using.

My simplest code:
```
require_once "lib/vendor/autoload.php";
$factory = new HeadlessChromium\BrowserFactory();
$browser = $factory->createBrowser(["headless" => false, "keepAlive" => true ]);
```

But when the script ends, the browser closes.

I've also tried with /snap/bin/chromium but get the same result.

**Comment by enricodias** [comment]
The other issues here are referring to the package `symfony/process` that this lib uses to create the chrome process. The keepAlive option should work. Are you testing it in linux?

**Comment by andreas-globi** [comment]
Yes, this is on Ubuntu 22.04. I tried both with the default installed Chrome, and with specifying Chromium from a snap install. Same behaviour with both. On script completion the browser closes.

I also tried from a CLI script, a PhpUnit test, and from PsySh directly.

PhpUnit gave some extra info: Process finished with exit code 129 (interrupted by signal 1:SIGHUP)

**Comment by netizen-ais** [comment]
I've this issue as well:

`ais@lynx:~/test> php -a
Interactive shell

php > include "vendor/autoload.php";
php > $browserFactory = new \HeadlessChromium\BrowserFactory("chromium-browser");
php > $browser = $browserFactory->createBrowser(['headless' => false, 'keeoAlive' => true, "debugLogger" => "php://stdout"]);
[2025-08-28T15:04:59.355918+00:00] chrome.DEBUG: process: initializing [] []
[2025-08-28T15:04:59.356489+00:00] chrome.DEBUG: process: using directory: /tmp/chromium-php-Z8wDxU [] []
[2025-08-28T15:04:59.357560+00:00] chrome.DEBUG: process: starting process: 'chromium-browser' '--remote-debugging-port=0' '--remote-allow-origins=*' '--disable-background-networking' '--disable-background-timer-throttling' '--disable-client-side-phishing-detection' '--disable-hang-monitor' '--disable-popup-blocking' '--disable-prompt-on-repost' '--disable-sync' '--disable-translate' '--disable-features=ChromeWhatsNewUI' '--metrics-recording-only' '--no-first-run' '--safebrowsing-disable-auto-update' '--enable-automation' '--password-store=basic' '--use-mock-keychain' '--user-data-dir=/tmp/chromium-php-Z8wDxU' [][]
[2025-08-28T15:04:59.358539+00:00] chrome.DEBUG: process: waiting for 30 seconds for startup [] []
[2025-08-28T15:04:59.531597+00:00] chrome.DEBUG: process: chrome output:DevTools listening on ws://127.0.0.1:39639/devtools/browser/20c9b193-d213-48ba-b142-0bf92cf93452 [] []`

Then I press ^D and Chromium closes:

`php >
php > ^D[2025-08-28T15:05:07.703851+00:00] chrome.DEBUG: process: killing chrome [] []
[2025-08-28T15:05:07.703914+00:00] chrome.DEBUG: process: trying to close chrome gracefully [] []
[2025-08-28T15:05:07.703942+00:00] chrome.DEBUG: socket(1): → sending data:{"id":2,"method":"Browser.close","params":{}} [] []`

I've seen that the backtrace is `ProcessAwareBrowser->kill()` is only called by `ProcessAwareBrowser->close()`
